support-frontend/assets/helpers/page/commonActions.js

```diff
@@ -18,7 +13,6 @@ export type SetCountryAction = { type: 'SET_COUNTRY_INTERNATIONALISATION', count
 export type Action =
   | SetCountryAction
   | { type: 'SET_EXISTING_PAYMENT_METHODS', existingPaymentMethods: ExistingPaymentMethod[] }
-  | { type: 'SET_TRACKING_CONSENT', trackingConsent: ThirdPartyTrackingConsent }
```

Comment:
It's quite nice to have this just within the consent form rather than the global state I think